src/WebBluetoothDeviceAdapter.js

```diff
@@ -54,6 +55,38 @@ export default class WebBluetoothDeviceAdapter {
       console.error(err);
     }
 
+    // Some platforms (Android) require us to tickle the command char to ensure it is ready
+    const _primeCommandChar = async () => {
+      const ch = this.deviceCommand;
```

Comment:
What does `ch` mean here? Character? Is there a better word that makes this more understandable?